src/project_configuration.h

```diff
@@ -21,13 +21,14 @@ class Configuration {
 
   class Wifi {
   public:
+    bool active;
```

Comment:
please move the active variable to the list some lines below.

Reply:
Adjusted formatting of class WiFi like class  APRS_IS